lib/tag_parser.rb

```diff
@@ -0,0 +1,54 @@
+
+module TagParser
+
+
+  def self.parse_tag(string)
+
+    type = string.match /<(\w+)/
+    type = type[1] if type
+    tag = Tag.new(type)
+
+    string.scan /\s(\w+)\s*=\s*['|"](.+?)['|"]/ do |key, value|
```

Comment:
I wouldn't have noticed but my text editor told me there was an "ambiguous argument" when not putting parentheses around the regex. I actually don't think it is a problem, just something that came up when I downloaded.